services/vpc_manager/src/main/java/com/futurewei/alcor/vpcmanager/entity/NetworkVxlanRange.java

```diff
@@ -52,8 +52,11 @@ public Long allocateKey() throws Exception {
     }
 
     public void release(Long key) throws Exception {
+//        if (allocated.get(key) == null) {
+//            throw new NetworkKeyAllocNotFoundException();
+//        }
         if (allocated.get(key) == null) {
-            throw new NetworkKeyAllocNotFoundException();
+            return;
```

Comment:
The question here is: is there any scenario that may need to throw an exception? If the answer is positive or likely, we would need to handle this differently. 

One option is throw NetworkKeyAllocNotFoundException here (no change), and handle the exception differently based on requirement. 
- In this fix, create a new TryReleaseXXX method which catches the exception but don't re-throw, only log info. 
- For other scenarios that requires to see the exception, continue to re-throw.

Reply:
Yeah that's what I think ~ when we could locate the issue, I will modify the code as you suggest